modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/sql/DistributionZoneSqlDdlParserTest.java

```diff
@@ -111,19 +112,19 @@ public void createZoneWithOptions() {
         assertThatZoneOptionPresent(optList, IgniteSqlZoneOptionEnum.DATA_NODES_AUTO_ADJUST, 1);
 
         expectUnparsed(createZone, "CREATE ZONE \"TEST_ZONE\" WITH "
-                + "REPLICAS = 2, "
-                + "PARTITIONS = 3, "
-                + "DATA_NODES_FILTER = '(\"US\" || \"EU\") && \"SSD\"', "
-                + "AFFINITY_FUNCTION = 'test_Affinity', "
-                + "DATA_NODES_AUTO_ADJUST = 1, "
-                + "DATA_NODES_AUTO_ADJUST_SCALE_UP = 2, "
-                + "DATA_NODES_AUTO_ADJUST_SCALE_DOWN = 3");
+                + "\"REPLICAS\" = 2, "
+                + "\"PARTITIONS\" = 3, "
+                + "\"DATA_NODES_FILTER\" = '(\"US\" || \"EU\") && \"SSD\"', "
+                + "\"AFFINITY_FUNCTION\" = 'test_Affinity', "
+                + "\"DATA_NODES_AUTO_ADJUST\" = 1, "
+                + "\"DATA_NODES_AUTO_ADJUST_SCALE_UP\" = 2, "
+                + "\"DATA_NODES_AUTO_ADJUST_SCALE_DOWN\" = 3");
     }
 
     /**
      * Parse CREATE ZONE WITH unknown option.
      */
-    @Test
+    @Disabled("https://issues.apache.org/jira/browse/IGNITE-19261")
```

Comment:
Please add a description to the issue that you've created. It's empty now

Reply:
Done